src/cascadia/WindowsTerminal/NonClientIslandWindow.cpp

```diff
@@ -113,8 +118,53 @@ void NonClientIslandWindow::OnSize(const UINT width, const UINT height)
         _rootGrid.Arrange(finalRect);
     }
 
+    // I'm not sure that HWND_BOTTOM is any different than HWND_TOP for us.

```

Comment:
I don't understand what this comment is supposed to mean.

Reply:
Oh mainly just that both those values do the same thing, which doesn't _really_ make sense, so there's not really a reason one was picked over the other